### ISSUE TITLE
Simulation Mode

### ISSUE THREAD

**Issue by sb2483** [SigPloiter/SigPloit#230]
I am doing a presentation about SMS hijacking and want to do a demo. I found SigPloit and it caught my attention. I have set up a Kali VM with static IP 192.168.56.102 and I have SigPloit running on it. I'd like to understand how to run a simulation to get output similar to the eighth page of the document at http://cysecure.org/560/online/project/sigploit_brianSteiner_joyGeorge_louisEscoto_emmanuelSefa.pdf, i.e., I'd like to simulate the interception of an SMS message. I think something has changed since that PDF file was made though because there doesn't seem to be anything in Testing/Server/Attacks/Interception that creates the ***      Intercepting          *** banner.

I have a second Kali VM with static IP 192.168.56.101 and both VMs can ping each other, in case I need a second VM (I got the impression I do from reading https://github.com/SigPloiter/SigPloit/issues/113). 

How do I perform a simple simulation of an SMS interception? Thanks for any help.

**Comment by EmmanuelTom** [comment]
Hello @sb2483  have you use the hackrf hardware?

**Comment by sb2483** [comment]
No I haven't but I know what you're talking about. I don't have $300 to
drop on it though. Is it needed to run simulation mode?

**Comment by EmmanuelTom** [comment]
I think is because of the absence of the hardware, what do you think?

**Comment by sb2483** [comment]
I don't think so but it could be. I can't imagine why hardware would be
needed for a simulation, but there's no documentation for how to stimulate
a client so I truly have no idea. Some simple instructions for "how to be a
client" when a stimulated server is running would be ideal.

**Comment by EmmanuelTom** [comment]
Okay, I have a project am working on also, will keep you posted then

**Comment by ShitalBorganve** [comment]
@sb2483 Did the issue resolved..??

**Comment by sb2483** [comment]
No, no progress has been made in this issue. To recap, I do have the
simulated server running through the UpdateLocation.jar. I see in the
parameters there is a "hello world" test message, but I don't know how to
"send" it so it shows up in the "intercepted" output on the server.

**Comment by zeroplus4** [comment]
Hackrfone not working properly, I suggest using live parameters for it

Sent from ProtonMail mobile

-------- Original Message --------
